frontend/portals/TabBarItem/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from 'glamor';
+import { useNavigation } from '@shopgate/engage/core';
+import isIOSTheme from '@shopgate-ps/pwa-extension-kit/env/helpers/isIOSTheme';
+import TopHat from '../../icons/TopHat';
+import { showSwellNavigation, swellButtonText, swellRewardsPath } from '../../config';
+
+const tabItemIconStyle = css({
+  height: 24,
+  width: 24,
+  flexGrow: 1,
+  marginRight: 'auto',
+  marginLeft: 'auto',
+  marginBottom: '2px',
+});
+
+/**
+ * TabBarItem.
+ * @param {Function} TabBarAction Tab Bar Action function.
+ * @param {string} path path to navigate to.
+ * @returns {JSX}
+ */
+const TabBarItem = ({ TabBarAction, path }) => {
+  const { push } = useNavigation();
```

Comment:
not needed because you get already `historyPush` from the props

Reply:
Cool, removed and fixed. Thanks